examples/intersection_observer/README.md

```diff
@@ -0,0 +1,11 @@
+# Intersection Observer Example
+
+This example uses the [Intersection Observer V1 API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) through web-sys.
+
+The intersection observer is created. We tell it to watch the red box by giving the element reference to the observer.
+When the red box is entirely in view, the label "Is intersecting entirely" will display a value of `true`.
```

Comment:
What do you think about something like _"Is completely visible"_  instead of _"Is intersecting entirely"_?

Reply:
Good idea!